### ISSUE TITLE
Every `make verify` reads the suite of an agent worktree standing under `.claude/worktrees/`

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#589]
### What happened?

`make verify` runs vitest over the whole tree, and `vitest.config.ts` excludes `tmp/**` beside vitest's own defaults. `.claude/` is excluded from Git per clone and is one directory for every worktree, a symlink at the main checkout's, and Claude Code puts the worktree it makes for an agent under `.claude/worktrees/<agent>/` — a full checkout of the repository, tests and all, on whatever commit that agent is working on.

So while such a worktree exists, every `make verify` in every checkout reads that agent's suite as its own, twice over where the two trees agree and red wherever that agent's work is mid-way. Measured on 2026-09-05 in the `url-in-text` worktree while `.claude/worktrees/agent-acd8b357090c4ac48` stood on `f17bb17`:

```shell
$ make verify
 Test Files  9 failed | 586 passed (595)
      Tests  24 failed | 20023 passed | 2 skipped (20049)
 FAIL  .claude/worktrees/agent-acd8b357090c4ac48/lib/rules/max-empty-lines/index.test.ts > …
 FAIL  .claude/worktrees/agent-acd8b357090c4ac48/scripts/harness/cache.test.ts > …

$ vitest run --exclude '.claude/**'
 Test Files  297 passed (297)
      Tests  10015 passed | 1 skipped (10016)
```

The same run half an hour earlier, before that worktree was made, was green at 297 files — so a branch's `make verify`, and the `pre-push` hook that stands on it, turns red on work that is nobody's on the branch, and the tree stamp `scripts/verified.ts` writes is refused for it.

### What should happen?

`make verify` reads the tests of the checkout it runs in and nothing under `.claude/`, which holds no file of the repository's: add `.claude/**` to the `exclude` of `vitest.config.ts` beside `tmp/**`. Whether the `related` run of the `pre-commit` hook and `oxlint` walk into that directory as well is worth measuring in the same change.
